plugins/available/alias-completion.plugin.bash

```diff
@@ -63,14 +63,17 @@ function alias_completion {
             local compl_func="${new_completion/#* -F /}"; compl_func="${compl_func%% *}"
             # avoid recursive call loops by ignoring our own functions
             if [[ "${compl_func#_$namespace::}" == $compl_func ]]; then
+                if [[ $compl_func == _filedir_xspec ]]; then
+                    compl_func_args=$alias_cmd
```

Comment:
Does it make sense to define the `compl_func_args` variable with an empty default value before the `if` statement? Just being cautious here, to make sure we don't run into any _undefined variable_ issues later...

Reply:
I've added the definition for `compl_func_args` with empty default.